http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -409,7 +409,7 @@ object Boot extends IOApp {
         F.delay(CaffeineCache[F, UserEmailAndProject, Option[String]](underlyingPetTokenCache))
       )(s => F.delay(s.close))
 
-      samDao = HttpSamDAO[F](Retry(retryPolicy)(httpClient), httpSamDaoConfig, petTokenCache)
+      samDao = HttpSamDAO[F](httpClientWithRetryAndLogging, httpSamDaoConfig, petTokenCache)
```

Comment:
added logging becuz I'm seeing lots of `"Max wait queue for limit of 256 for https://sam.dsde-prod.broadinstitute.org:443 reached, not scheduling."` in prod